pkg/config/env/ipc_address.go

```diff
@@ -38,9 +39,19 @@ func IsLocalAddress(address string) (string, error) {
 
 // GetIPCAddress returns the IPC address or an error if the address is not local
 func GetIPCAddress(cfg pkgconfigmodel.Reader) (string, error) {
-	address, err := IsLocalAddress(cfg.GetString("ipc_address"))
+	var key string
+	// ipc_address is deprecated in favor of cmd_host, but we still need to support it
+	// if it is set, use it, otherwise use cmd_host
+	if cfg.IsSet("ipc_address") {
```

Comment:
Could we add a cipule of tests around this logic?

Reply:
Done